src/includes/blog-article.njk

```diff
@@ -0,0 +1,24 @@
+<li class="blog-article">
```

Comment:
Any reason for using tabs in this file? 🤔

Reply:
mm nope, I'll fix that.